src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -746,7 +746,14 @@ private Voucher withRequiredFields(Voucher voucher, Invoice invoice) {
     return expenseClassRetrieveService.getExpenseClasses(expenseClassIds, new RequestContext(ctx, okapiHeaders))
                                .thenApply(expenseClasses -> expenseClasses.stream().collect(toMap(ExpenseClass::getId, Function.identity())))
                                .thenApply(expenseClassByIds ->
-                                 fundDistrs.stream().collect(
+                                 fundDistrs.stream()
+                                   .peek(fd -> {
```

Comment:
`peek` is documented to be mainly for debugging purposes. Please use`map` operation instead of peek